src/h3lib/include/stackAlloc.h

```diff
@@ -0,0 +1,44 @@
+#ifndef STACKALLOC_H
```

Comment:
Please also include a `@file`/`@brief` comment describing the purpose of this file.

Reply:
Good idea